### DISCUSSION TITLE
DOUBLE WORDS AND PHRASES - PUNCTUATION

### DISCUSSION THREAD

**Discussion by Nevergiveupvp** [deepgram/community#627] [General help] [answered]
I can't seem to find a solution to this, but when we load a pre recorded audio the text comes out with repeated words as:  I went went went to the bank, whereas it should be I went, went, went to the bank.  There are no commas between the repeated words or phrases.  Can anyone assist is this a deepgram request or a developer fix on client's end.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by Nevergiveupvp** [comment]
You you are 
 
Transcribed by deepgram.

**Comment by jpvajda** [comment, marked as answer]
@Nevergiveupvp 👋  As far as I know there isn't a way yet to prompt the Deepgram STT models to use commas in certain places, I might be wrong here so I'll leave this thread open to see if anyone else has a solution to this challenge.
